app/models/user.rb

```diff
@@ -16,6 +16,18 @@ def user_repos(count = 50)
     repos[0...count]
   end
 
+  def user_followers(count = 50)
+    if github_token
+      GithubSearch.new.followers(github_token)[0...count]
+    end
+  end
+
+  def user_following(count = 50)
+    if github_token
+      GithubSearch.new.following(github_token)[0...count]
+    end
+  end
+
```

Comment:
Glad you made these these methods. I like this approach of being able to call these methods off the user objects.